netshare/utils/field.py

```diff
@@ -191,6 +206,53 @@ def getOutputDim(self) -> int:
         return 2 * self.num_bits  # type: ignore
 
 
+class Word2VecField(Field):
```

Comment:
can you add typing here?
Maybe I'll add it as a constraint in mypy on my next PR :)